src/modules/fancyzones/FancyZonesLib/WindowUtils.cpp

```diff
@@ -322,18 +322,27 @@ void FancyZonesWindowUtils::SizeWindowToRect(HWND window, RECT rect) noexcept
         ::GetWindowPlacement(window, &placement);
     }
 
-    // Do not restore minimized windows. We change their placement though so they restore to the correct zone.
-    if ((placement.showCmd != SW_SHOWMINIMIZED) &&
-        (placement.showCmd != SW_MINIMIZE))
+    bool isVisible;
+    isVisible = IsWindowVisible(window);
+    if (!isVisible)
```

Comment:
I don't see why this can't be done in one go:
```suggestion
    if !IsWindowVisible(window)
```
(return type of method is also bool)